cli/cmd/encore/app/create_form.go

```diff
@@ -289,51 +305,70 @@ func selectTemplate(inputName, inputTemplate string) (appName, template string)
 }
 
 func loadTemplates() tea.Msg {
-	// Get the list of templates from GitHub
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
-	defer cancel()
-	url := "https://raw.githubusercontent.com/encoredev/examples/main/cli-templates.json"
-	if req, err := http.NewRequestWithContext(ctx, "GET", url, nil); err == nil {
-		if resp, err := http.DefaultClient.Do(req); err == nil {
-			if data, err := io.ReadAll(resp.Body); err == nil {
-				if data, err = hujson.Standardize(data); err == nil {
-					var items []templateItem
-					if err := json.Unmarshal(data, &items); err == nil && len(items) > 0 {
-						return loadedTemplates(items)
+	// Load the templates.
+	templates := (func() []templateItem {
+		// Get the list of templates from GitHub
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+		defer cancel()
+		url := "https://raw.githubusercontent.com/encoredev/examples/main/cli-templates.json"
+		if req, err := http.NewRequestWithContext(ctx, "GET", url, nil); err == nil {
+			if resp, err := http.DefaultClient.Do(req); err == nil {
+				if data, err := io.ReadAll(resp.Body); err == nil {
+					if data, err = hujson.Standardize(data); err == nil {
+						var items []templateItem
+						if err := json.Unmarshal(data, &items); err == nil && len(items) > 0 {
+							return items
+						}
 					}
 				}
 			}
 		}
+
+		// Return a precompiled list of default items in case we can't read them from GitHub.
+		return []templateItem{
+			{
+				ItemTitle: "Hello World",
+				Desc:      "A simple REST API",
+				Template:  "hello-world",
+			},
+			{
+				ItemTitle: "Uptime Monitor (TypeScript)",
+				Desc:      "Microservices, SQL Databases, Pub/Sub, Cron Jobs",
+				Template:  "uptime-ts",
+			},
+			{
+				ItemTitle: "Uptime Monitor (Go)",
+				Desc:      "Microservices, SQL Databases, Pub/Sub, Cron Jobs",
+				Template:  "uptime",
+			},
+			{
+				ItemTitle: "GraphQL",
+				Desc:      "GraphQL API, Microservices, SQL Database",
+				Template:  "graphql",
+			},
+			{
+				ItemTitle: "URL Shortener",
+				Desc:      "REST API, SQL Database",
+				Template:  "url-shortener",
+			},
+			{
+				ItemTitle: "Empty app",
+				Desc:      "Start from scratch (experienced users only)",
+				Template:  "",
+			},
+		}
+	})()
+
+	// If we're not running the Encore beta (with TypeScript support),
+	// filter out any TypeScript items.
+	haveTypeScript := version.Channel != version.GA
+	if !haveTypeScript {
+		templates = fns.Filter(templates, func(t templateItem) bool {
+			return !strings.Contains(strings.ToLower(t.ItemTitle), "typescript")
```

Comment:
Guess this is a temporary thing. Otherwise it might be useful to introduce a language field to the template items

Reply:
Yeah, just temporary